src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -200,11 +340,21 @@ public static class Sse
         /// </summary>
         public static unsafe void Store(float* address, Vector128<float> source) => Store(address, source);
 
+        /// <summary>
+        /// void _mm_store_ss (float* mem_addr, __m128 a)
+        /// </summary>
+        public static unsafe void StoreScalar(float* address, Vector128<float> source) => StoreScalar(address, source);
+
         /// <summary>
         /// __m128d _mm_sub_ps (__m128d a, __m128d b)
```

Comment:
note: the files have several typos, such as this, where the wrong type is used in the intrinsic

Reply:
Good catch. For these comment typos, we can fix them later that do not impact the CoreCLR/CoreFX interface.